src/Microsoft.Identity.Client/IPublicClientApplication.cs

```diff
@@ -42,6 +39,13 @@ namespace Microsoft.Identity.Client
     /// </summary>
     public partial interface IPublicClientApplication : IClientApplicationBase
     {
+        /// <summary>
+        /// Tells if the application can the system web browser, therefore getting single-sign-on with web applications.
+        /// Today, this is the case on Xamarin.Android, Xamarin.iOS, and the system web view is used by default.
```

Comment:
This sentence is a bit misleading, because on Android there's all that logic with custom tabs. May smth like "By default, MSAL will try to use a system browser on the mobile platforms if it is available. "

Reply:
#fixed